### ISSUE TITLE
[feature request] Use force feedback as input for impulse trigger?

### ISSUE THREAD

**Issue by djbiti1** [araghon007/X1nput#41]
I am not sure if it would be possible, but maybe for racing games, the force feedback could be used as input for impulse trigger, while keeping the normal vibration as is?

**Comment by araghon007** [comment]
By force feedback, do you mean feedback used for racing wheels?

**Comment by djbiti1** [comment]
Exactly, the input that tells the wheel how much force it should opose you. I assume this is sent to xinput so maybe it can be translated to the impulse trigger somehow?

**Comment by araghon007** [comment]
I could look into that, but I believe the game would have to think there is a wheel present, and I'm not sure how well that would work with controller input. Would definitely be nice though
